src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -88,11 +89,15 @@ public void run() {
                         new TkFork(
                             new FkCommand(
                                 "/notifications",
-                                    new TkNotifications(this.data)
+                                new TkNotifications(this.data)
                             ),
                             new FkCallbackQuery(
                                 Pattern.compile("click:notification#(?<tid>[A-Za-z0-9]+)"),
-                                    new TkThread(this.data)
+                                new TkThread(this.data)
+                            ),
+                            new FkCallbackQuery(
+                                Pattern.compile("click:notification.close.+"),
```

Comment:
@olenagerasimova this is strange to have this pattern here and another one in the class... find a way to have this concern not distributed in multiple classes. Add a todo with some starting idea for example, maybe by having one class implementing `Fork` and containing both the current code of `TkCloseIssue` and the creation of the `FkCallbackQuery`?